### PR TITLE
Require users to call cmu_graphics.run()

### PR DESCRIPTION
**Pull request by austin-schick**
Replaces `cmu_graphics.loop()` with `cmu_graphics.run()`
Doesn't attempt to automatically run the mainloop for a user's app
Fixes the issue where an interactive console would be created even when an error happened before `cmu_graphics.loop()` was called
Prints a help message when shapes are created by `cmu_graphics.run()` was never called

### CODE REVIEW

**Comment by austin-schick** [comment]
I think we should check in with the fulltimers to see what they think about loop vs run vs runApp vs something else

File: cmu_graphics/cmu_graphics.py [hunk 1]
```diff
@@ -679,5 +685,18 @@ def print_debug_info():
...
+
+# Checks to see if a user created shapes but did not call
+# cmu_graphics.run()
+def check_for_exit_without_run():
+    global SHAPES_CREATED, MAINLOOP_RUN
+
+    # The app's top level group is created even if the user creates no
+    # shapes on their own
+    if SHAPES_CREATED > 1 and not MAINLOOP_RUN:
+        print("To run your animation, add cmu_graphics.run() to the bottom of your file")
```
Comment (schmave) [inline cmu_graphics/cmu_graphics.py:699]: Maybe add some ASCII art or a couple newlines before this statement so that it is more visible?

**Comment by schmave** [comment]
Since it looks like we're going with `.run()`, you could update the README too

**Comment by austin-schick** [comment]
@schmave I think this is ready for another look. The build is failing, but only because of a weird segfault in the Python3.7 pip tests

**Pull request merged**